threescale/src/utils.rs

```diff
@@ -1,6 +1,20 @@
+use crate::structs::Period;
 use crate::structs::{Application, ThreescaleData};
+use threescalers::response::Period as ResponsePeriod;
 
 // Perform metrics update based on threescale specific logic
 pub fn update_metrics(_new_hits: &ThreescaleData, _application: &mut Application) -> bool {
     true
 }
+
+pub fn period_from_response(res_period: &ResponsePeriod) -> Period {
+    match res_period {
+        ResponsePeriod::Minute => Period::Minute,
+        ResponsePeriod::Hour => Period::Hour,
+        ResponsePeriod::Day => Period::Day,
+        ResponsePeriod::Week => Period::Week,
+        ResponsePeriod::Month => Period::Month,
+        ResponsePeriod::Year => Period::Year,
+        ResponsePeriod::Eternity => Period::Eternity,
+    }
+}
```

Comment:
Can you try to implement `from` trait as it was mentioned by Alex in my last (or 2nd last PR?). I had issues with dependency conflicts back then used this instead. Now, that it can be placed into threescale, please convert this into a `from` trait for `Period` Enum. Thanks!

Reply:
fixed